_plugins/content_typography.rb

```diff
@@ -0,0 +1,28 @@
+module Kramdown
+  module Parser
+    class Kramdown
+      prepend(Module.new do
+        def new_block_el(type, *args)
+          el = super(type, *args)
+          add_css_class!(el, 'usa-list') if type == :ul
+          el
+        end
+
+        def add_link(el, *args)
+          add_css_class!(el, 'usa-link') if el.type == :a
+          super(el, *args)
+        end
```

Comment:
glad we're able to combine these into the same plugin!

Reply:
Yeah, I'm increasingly liking the idea of a "USWDS Jekyll" plugin to auto-decorate parsed Markdown content with relevant USWDS classes. But not sure it's worth that much investment in Jekyll at this point. I'm hopeful the idea is portable enough wherever we land on static site generators.